src/openzaak/management/commands/generate_data.py

```diff
@@ -250,6 +265,7 @@ def generate_catalogi(self):
         # zaaktype - 100
         zaaktypen = ZaakTypeFactory.build_batch(
             self.zaaktypen_amount,
+            selectielijst_procestype=random.choice(list(self.sl_result_mapping.keys())),
```

Comment:
Maybe I understand wrong, but I thought that the procestype referenced by the zaaktype should be from this endpoint: https://selectielijst.openzaak.nl/api/v1/procestypen and not from the resultaten endpoint?

Reply:
`Resultaat` resource in Selectielijst API has a FK to `Procestype` 
So here we request `resultaten` endpoint, retrieve `procestype` and `url` attributes and create a dictionary with them. 

The idea is to add some random `procestypes` to zaaktypen and then to add related `resultaten` to the zaken

Reply:
Ahh I see!